examples/ai-core/src/generate-text/azure-image.ts

```diff
@@ -0,0 +1,39 @@
+import { azure } from '@ai-sdk/azure';
+import { generateText } from 'ai';
+import 'dotenv/config';
+import fs from 'node:fs';
+
+async function main() {
+  const imageData = fs.readFileSync('/Desktop/sonny-angel.jpg');
+  const imageBase64_string = imageData.toString('base64');
+
+  const { text, usage } = await generateText({
+    model: azure('v0-gpt-35-turbo'), // use your own deployment
+    messages: [
+      {
+        role: 'user',
+        content: [
+          { type: 'text', text: 'Describe the image in detail.' },
+          {
+            type: 'image',
+            image: imageBase64_string,
+            // Even if you mislabel the mime type, internally the AI SDK will detect and use the correct mime type:
+            mimeType: 'image/svg+xml',
```

Comment:
is possible to just leave out the mimetype (this should be the recommended way as it has the best dx and we auto-recognize images)